shell/platform/android/io/flutter/plugin/editing/TextEditingDelta.java

```diff
@@ -0,0 +1,145 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugin.editing;
+
+import androidx.annotation.VisibleForTesting;
+import io.flutter.Log;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+/// A representation of the change that occured to an editing state, along with the resulting
+/// composing and selection regions.
```

Comment:
Do we explicitly differentiate between text content changes and selection/composing region index only changes? However it is handled, it should be documented.

Reply:
Never mind, that is under the umbrella of TextEditingDeltaNonTextUpdate

Reply:
On the framework we do, but on the engine there is no differentiation. This differentiation is documented on the framework side, should it also have something here?